templates/_helpers.tpl

```diff
@@ -0,0 +1,17 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
Do we really need this?

Reply:
I mean the all helper

Reply:
it generates the name, truncating in case it is long and adding hyphens. I borrowed from the incubator chart because I liked